django_forest/utils/schema/validations.py

```diff
@@ -0,0 +1,74 @@
+from django.core.validators import MaxLengthValidator, MinLengthValidator, MaxValueValidator, MinValueValidator, \
+    RegexValidator
+
+
+VALIDATORS = {
+    'max_length': {
+        'type': 'is shorter than',
+        'message': 'Ensure this value has at most %(limit_value)d characters'
+    },
+    'min_length': {
+        'type': 'is longer than',
+        'message': 'Ensure this value has at least %(limit_value)d characters'
+    },
+    'max_value': {
+        'type': 'is less than',
+        'message': 'Ensure this value is less than or equal to %(limit_value)s characters'
+    },
+    'min_value': {
+        'type': 'is greater than',
+        'message': 'Ensure this value is greater than or equal to %(limit_value)s characters'
+    },
+}
+
+
+def handle_validator(validator, f):
+    if isinstance(validator, MaxLengthValidator) \
+            or isinstance(validator, MinLengthValidator) \
+            or isinstance(validator, MaxValueValidator) \
+            or isinstance(validator, MinValueValidator):
```

Comment:
Do you need to check against classes, or is this enough?
```suggestion
    if validator.code in VALIDATORS:
```